.changeset/gold-olives-rule.md

```diff
@@ -0,0 +1,7 @@
+---
+'@mermaid-js/examples': minor
+'mermaid': minor
+---
+
+feat: Add examples for diagrams in the `@mermaid-js/examples` package
+feat: Add `getDiagramData` to `mermaid`, which returns all the registered diagram IDs in mermaid
```

Comment:
I think this should be split into two separate changesets.

Otherwise, the following message would appear both in the `@mermaid-js/examples` release notes/`CHANGELOG.md` and the `mermaid` release notes.

But I think you'd rather want to have the first line only in the `@mermaid-js/examples` and the second line only in the `mermaid` release notes.